homeassistant/components/renault/quality_scale.yaml

```diff
@@ -47,7 +47,11 @@ rules:
   docs-supported-functions: todo
   docs-troubleshooting: todo
   docs-use-cases: todo
-  dynamic-devices: todo
+  dynamic-devices:
+    status: exempt
+    comment: |
+      Adding/removing a vehicle is a rare occurrence, and we don't want to
+      waste valuable hits on Renault API, which is rate-limited
```

Comment:
This seems like odd reasoning to me: A rare occurrence + worrying about rate-limiting.

If I might suggest an alternative, could you trigger a config entry reload if the change is detected?

Reply:
There is no way to know without making a specific API "list vehicles" request

Reply:
@frenck are you suggesting that I should make an API call at regular interval to see if there are new vehicles?
If we are making a "list vehicles" call every few minutes we would need to reduce the interval for more important data calls.

I guess we could make a call every 24h, but seems a lot of work for very little benefit.